kubeconform/analyse.py

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env python3
+#
+# This script iterates over all kustomize overlays under specified
+# directory, renders the resulting manifests and performs a kubeconform check on them
+import subprocess
+import tempfile
+import argparse
+import os.path
+import sys
+from pathlib import Path
+
+
+parser = argparse.ArgumentParser()
+parser.add_argument('analyse_dir', nargs='?', default='.')
+parser.add_argument('--schema-location', action='append')
```

Comment:
and in that case why not also passing it the same way to our own cli?

Reply:
AFAIK `argparse` only allows `--` for long options.

Reply:
should work just fine 🤔

Reply:
Indeed it works. I learn something new every day :)